src/pages/CreateEventPage/components/FormFields/styled.ts

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
+
+export const InputText = styled.div`
+  margin: 0 2.5vw;
+  width: 300px;
+  margin: 10px;
+
+  input {
+    width: 100%;
+  }
+  
+  textarea {
+    width: 100%;
+    height: 100px;
+  }
+
+  select {
+    width: 100%;
+  }
```

Comment:
Implicitly styling descendants like this is an antipattern. You should try to create a style component for each individual element instead, which can then be used as children of this component. This is to avoid having to deal with issues regarding specificity and selector precedence later down the road. Also, implicitly inheriting basic styling like this is never a good idea, as you never know if a future field component might need a different look - which would force it to override this implicit styling.